e2e/cypress/tests/support/ldap_server_commands.ts

```diff
@@ -110,3 +127,48 @@ mail: ${user.email}
 userPassword: Password1
 `;
 }
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+
+            /**
+             * modify LDAP users
+             * @param {string} filename
+             * @returns void
+             */
+            modifyLDAPUsers: typeof modifyLDAPUsers;
+
+            /**
+             * reset LDAP users
+             * @returns void
+             */
+            resetLDAPUsers(): void;
+
+            ldapAdd: typeof ldapAdd;
+            ldapModify: typeof ldapModify;
+
+            /**
+             * update LDAP users
+             * @param {string} prefix
+             * @param {LDAPUserArg} user
+             * @returns ChainableT<any>
+             */
+            createLDAPUser({prefix: string, user: LDAPUserArg}): ChainableT<any>;
```

Comment:
Please match return type with definition

Reply:
I guess it's <JQuery>.